media/flv.ksy

```diff
@@ -0,0 +1,63 @@
+meta:
+  id: flv
+  title: Adobe Flash Video File Format
+  license: CC0-1.0
+  ks-version: 0.9
+  encoding: utf-8
+  endian: be
+doc-ref: https://wwwimages2.adobe.com/content/dam/acom/en/devnet/flv/video_file_format_spec_v10_1.pdf
+seq:
+  - id: preheader
+    type: preheader
+  - id: rest_of_header
+    size: preheader.len_header - preheader._sizeof
+  - id: previous_tag_size0
+    type: u4
+    valid: 0
+  - id: tags
+    type: tag_and_size
+    repeat: eos
+types:
+  preheader:
+    seq:
+      - id: magic
+        contents: 'FLV'
+      - id: version
+        type: u1
+        valid: 1
+      - id: flags
+        type: u1
+      - id: len_header
+        type: u4
+  tag_and_size:
+    seq:
+      - id: tag
+        type: tag
+      - id: previous_tag_size
+        type: u4
+        valid: tag.len_data + 11
```

Comment:
Make a subtype and use `sizeof`

Reply:
Why? The specification says that this is always + 11. It has never been anything else.

Reply:
Because contents of `tag` except of `data` is exactly 11 bytes, and `11 + len_data` is the len of `tag`. Also it may make sense to use `_io.pos`

Reply:
when reading the specifications I think that this is clearer (as it follows the specifications closer). Yes, it is hardcoded, but I don't see it as a problem.